src/backend/distributed/shared_library_init.c

```diff
@@ -154,7 +154,11 @@ _PG_init(void)
 	planner_hook = multi_planner;
 
 	/* register utility hook */
+#if (PG_VERSION_NUM >= 100000)
+	ProcessUtility_hook = multi_ProcessUtility10;
```

Comment:
Having 10 in the name will be a bit awkward for 11+, so maybe name the wrapper Pre10 or such?